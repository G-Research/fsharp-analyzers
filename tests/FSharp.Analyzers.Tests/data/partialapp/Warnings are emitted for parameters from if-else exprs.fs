module M

let myFunc x y = x + y

let _ = myFunc (if true then 1 else 0) // should warn
let _ = myFunc (if true then 1 else 0) (if true then 1 else 0) // should not warn
