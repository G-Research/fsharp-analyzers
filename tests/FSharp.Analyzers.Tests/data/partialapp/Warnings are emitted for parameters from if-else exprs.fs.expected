GRA-PARTAPP-001 | Warning | (5,8 - 5,14) | Partial application should not be used. | []
